vizro-core/tests/integration/conftest.py

```diff
@@ -0,0 +1,22 @@
+# ruff: noqa: F403, F405
+import os
+
+import chromedriver_autoinstaller_fix
+import pytest
+
+
+# Use monkeypatch as a session-scoped fixture.
+# Taken from https://github.com/pytest-dev/pytest/issues/363#issuecomment-1335631998.
+@pytest.fixture(scope="session")
+def monkeypatch_session():
+    with pytest.MonkeyPatch.context() as mp:
+        yield mp
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Actually I think what we should do is leave this stuff in `test_examples.py`, because it's only relevant for that file, and do a find and replace `session` to `module`.